src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/CollectionExerciseServiceImpl.java

```diff
@@ -102,6 +103,12 @@ public CollectionExerciseServiceImpl(
     return this.collectRepo.findBySurveyId(UUID.fromString(survey.getId()));
   }
 
+  @Override
+  public List<CollectionExercise> findCollectionExercisesBySurveyIdAndState(
+      UUID surveyId, CollectionExerciseState state) {
+    return collectRepo.findBySurveyIdAndState(surveyId, state);
```

Comment:
Codecov is complaining this is not covered in a test might also be worth adding one for it

Reply:
a unit test for a 1-liner seems excessive

Reply:
we're going to refactor away the stupid wrapper functions around repo methods

Reply:
I thought bout that too but just codecov being codecov but yeah if we're removing them wrapper functions thats cool 👍